controllers/deployment.go

```diff
@@ -133,6 +168,37 @@ func (r *DeploymentReconciler) Reconcile(ctx context.Context, application *skipe
 			}
 
 			volumeMounts[i+1] = corev1.VolumeMount{Name: volumes[i+1].Name, MountPath: file.MountPath}
+			VolNumber = i
+		}
+
+		if application.Spec.GCP != nil {
+
+			volumeMounts[VolNumber+1].Name = "gcp-ksa"
+			volumeMounts[VolNumber+1].MountPath = "/var/run/secrets/tokens/gcp-ksa"
+			volumeMounts[VolNumber+1].ReadOnly = true
+
+			Expiration := int64(172800)
```

Comment:
Can we make this esier to read by extracting a variable?
```suggestion
                        twoDays := int64(172800)
			Expiration := twoDays
```

Reply:
Changed names and converted casing